orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -102,6 +108,8 @@ class OperationsController {
   @Autowired(required = false)
   Front50Service front50Service
 
+  def jsonSlurper = new JsonSlurper()
```

Comment:
While this is a groovy class, I'd prefer to not increase reliance on Groovy-only classes, as it'll make things just that much more difficult to refactor this out of Groovy in the future. Use Jackson instead!

Reply:
Incorporated the change.